kythe/cxx/tools/kindex_tool_main.cc

```diff
@@ -83,6 +86,59 @@ void CanonicalizeHash(std::map<google::protobuf::string, size_t>* hashes,
       google::protobuf::string("hash" + std::to_string(inserted.first->second));
 }
 
+void DumpCompilationUnit(const std::string& path,
+                         kythe::proto::CompilationUnit* unit) {
+  using namespace google::protobuf::io;
```

Comment:
This doesn't have the semantics you want it to and is pretty error-prone (all of these symbols will be found as if they were at global scope in this function).  Please using-declare the entities you actually want.